__tests__/seed.test.js

```diff
@@ -0,0 +1,42 @@
+const app = require("../app.js");
+require("jest-sorted");
+const request = require("supertest");
+const db = require("../db/connection.js");
+const seed = require("../db/seeds/seed.js");
+const {
+  topicData,
+  userData,
+  articleData,
+  commentData,
+} = require("../db/data/test-data/index.js");
+
+beforeEach(() => seed({ topicData, userData, articleData, commentData }));
+afterAll(() => db.end());
+
+describe("/api/topics", () => {
+  test("GET:200 sends an array of topics to the client with the correct length and datatype", () => {
+    return request(app)
+      .get("/api/topics")
+      .expect(200)
+      .then((res) => {
+        const { topics } = res.body;
+        expect(topics.length).toBe(3);
+        topics.forEach((topic) => {
+          expect(typeof topic.description).toBe("string");
+          expect(typeof topic.slug).toBe("string");
+        });
+      });
+  });
+  test("GET:200 topics[0] should match given object", () => {
```

Comment:
This test is a little redundant. Have a look at `.objectContaining`, this could be a really good jest matcher to use instead to look at the properties and their values.